resource/graph_node.go

```diff
@@ -214,14 +214,20 @@ func (w *GraphNode) MarkedForRemoval() bool {
 	return w.markedForRemoval
 }
 
-// SetLastError sets the latest error on this node. This will
+// LogAndSetLastError sets the latest error on this node. This will
```

Comment:
```suggestion
// LogAndSetLastError logs and sets the latest error on this node. This will
```

Reply:
Thanks for catching that. I've incorporated locally to avoid commiting here + merging.